src/main/java/it/pagopa/pn/deliverypush/utils/HtmlSanitizerJsonParserDelegate.java

```diff
@@ -0,0 +1,32 @@
+package it.pagopa.pn.deliverypush.utils;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.util.JsonParserDelegate;
+import org.apache.commons.text.StringEscapeUtils;
+import org.owasp.html.PolicyFactory;
+
+import java.io.IOException;
+
```

Comment:
Commenta l'utilità di questa classe. Con casi d'uso. Esempio: utilizzata in... per

Reply:
Aggiunto javadoc